[2022-07-29 22:40:39,488] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataperf.plotter manual__2022-07-29T22:28:10.141456+00:00 [queued]>
[2022-07-29 22:40:39,497] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataperf.plotter manual__2022-07-29T22:28:10.141456+00:00 [queued]>
[2022-07-29 22:40:39,498] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 22:40:39,498] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-29 22:40:39,499] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 22:40:39,513] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): plotter> on 2022-07-29 22:28:10.141456+00:00
[2022-07-29 22:40:39,518] {standard_task_runner.py:52} INFO - Started process 707 to run task
[2022-07-29 22:40:39,521] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dataperf', 'plotter', 'manual__2022-07-29T22:28:10.141456+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/dataperfr.py', '--cfg-path', '/tmp/tmp864vdj3_', '--error-file', '/tmp/tmpfvc56zsx']
[2022-07-29 22:40:39,522] {standard_task_runner.py:80} INFO - Job 229: Subtask plotter
[2022-07-29 22:40:39,585] {task_command.py:371} INFO - Running <TaskInstance: dataperf.plotter manual__2022-07-29T22:28:10.141456+00:00 [running]> on host 5eb1407492fa
[2022-07-29 22:40:39,657] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dataperf
AIRFLOW_CTX_TASK_ID=plotter
AIRFLOW_CTX_EXECUTION_DATE=2022-07-29T22:28:10.141456+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-29T22:28:10.141456+00:00
[2022-07-29 22:40:39,658] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-29 22:40:39,659] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /opt/***/dags/dependency; python plotter.py']
[2022-07-29 22:40:39,668] {subprocess.py:85} INFO - Output:
[2022-07-29 22:40:41,212] {subprocess.py:92} INFO - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-lynotoql because the default path (/home/***/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-07-29 22:40:42,530] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-29 22:40:42,562] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dataperf, task_id=plotter, execution_date=20220729T222810, start_date=20220729T224039, end_date=20220729T224042
[2022-07-29 22:40:42,583] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-29 22:40:42,615] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
